src/View/Shortcodes/EmbedShortcodeProvider.php

```diff
@@ -89,7 +91,22 @@ public static function handle_shortcode($arguments, $content, $parser, $shortcod
         }
 
         // Process embed
-        $embed = $embed->getEmbed();
+        try {
+            $embed = $embed->getEmbed();
+        } catch (InvalidUrlException $e) {
+            $message = (Director::isDev()) ? $e->getMessage() : _t(__CLASS__.'.INVALID_URL', 'There was a problem loading the media.');
```

Comment:
```suggestion
            $message = (Director::isDev())
                ? $e->getMessage()
                : _t(__CLASS__.'.INVALID_URL', 'There was a problem loading the media.');
```

Line is currently a bit too long, I’m not sure why scrutinizer missed this... 🤔